### ISSUE TITLE
Too much 1RTT data in interop test against QUANT.

### ISSUE THREAD

**Issue by huitema** [private-octopus/picoquic#1312]
The Interop Runner complains that in a test against QUANT, Picoquic is sending too much 1RTT data. The report says:
```
2022-03-11 11:20:53,659 Using the client's key log file.
2022-03-11 11:20:54,116 Check of downloaded files succeeded.
2022-03-11 11:20:54,116 Using the client's key log file.
2022-03-11 11:20:54,605 0-RTT size: 10378
2022-03-11 11:20:54,606 1-RTT size: 5049
2022-03-11 11:20:54,606 Client sent too much data in 1-RTT packets.
```

Analysis of the [client qlog](https://interop.seemann.io/logs/2022-03-11T08:09/quant_picoquic/zerortt/client/qlog/be2eba773d124c29.client.qlog) shows the following issues:

* Sending a PMTUD packet (1500 bytes) when the client does not have much in the way of queued traffic
* Acknowledgements are padded to 55 bytes
* Connection Close packet is repeated multiple times, withut waiting for a 1 RTT timer.

**Comment by huitema** [comment]
The test does pass in some runs, e.g.:
```
2022-03-12 19:17:01,150 Using the client's key log file.
2022-03-12 19:17:01,752 Check of downloaded files succeeded.
2022-03-12 19:17:01,753 Using the client's key log file.
2022-03-12 19:17:02,397 0-RTT size: 10378
2022-03-12 19:17:02,397 1-RTT size: 4981
```
Two changes should make it pass all the time:

* Adopt the "delayed PMTUD" mode on client side, per PR #1315 
* Send fewer connection close packets in closing state, per Pr #1314 

Waiting for next run of interop runner to see the results.

**Comment by huitema** [comment]
Fixed in PR #1317
